recipes/ms/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+gcc -O3 -o ms ms.c streec.c rand1.c -lm
```

Comment:
Can you add gcc as a build requirement? I know this is not consistently done yet in Bioconda, but I think it does not hurt and might improve portability in some rare cases.